Server/serverSlate.go

```diff
@@ -23,24 +23,24 @@ type action struct {
 	Message string `json:"message"`
 }
 
-type point struct {
+type Point struct {
 	X int `json:"x"`
 	Y int `json:"y"`
 }
 
-type frameData struct {
-	CornerPoints []point `json:"cornerPoints"`
+type FrameData struct {
```

Comment:
You shouldn't need to make these public just for testing.  go unit tests have access to private stuff in the package they live in.

Reply:
Didn't knew about this, fixed!